selfdrive/controls/lib/latcontrol_curvature.py

```diff
@@ -0,0 +1,49 @@
+import math
+
+from cereal import log
+from selfdrive.controls.lib.latcontrol import LatControl, MIN_STEER_SPEED
+from selfdrive.controls.lib.pid import PIDController
+
+STEER_ANGLE_SATURATION_THRESHOLD = 2.5  # Degrees
+
+
+class LatControlCurvature(LatControl):
+  def __init__(self, CP, CI):
+    super().__init__(CP, CI)
+    self.pid = PIDController((CP.lateralTuning.pid.kpBP, CP.lateralTuning.pid.kpV),
+                             (CP.lateralTuning.pid.kiBP, CP.lateralTuning.pid.kiV),
+                             k_f=CP.lateralTuning.pid.kf, pos_limit=self.steer_max, neg_limit=-self.steer_max)
```

Comment:
```suggestion
                             k_f=CP.lateralTuning.pid.kf)
```

Let's leave it up to the PID defaults (practically unlimited). Any number seems arbitrary to me. 0.02 is 20 m/s^2 of lateral accel at 70 mph, and 1.0 (steer_max) is similar in that vein.